content/docs/gke/private-clusters.md

```diff
@@ -4,8 +4,23 @@ description = "How to create private GKE clusters"
 weight = 5
 +++
 
-This guide describes how to create private clusters with Kubeflow on Google 
-Kubernetes Engine (GKE).
+This guide describes how to secure Kubeflow using private GKE and [VPC service Controls](https://cloud.google.com/vpc-service-controls/docs/)
```

Comment:
Should we change the title of the page? It's currently "Creating Private Clusters" but perhaps it should be more generic, about securing your clusters or something. Also the `description` under the title.

Reply:
Done